src/js/binaryen.js-post.js

```diff
@@ -486,8 +486,6 @@ function wrapModule(module, self) {
   // need to make their own Literals, as the C API handles them by value,
   // which means we would leak them. Instead, this is the only API that
   // accepts Literals, so fuse it with Literal creation
-  var temp = _malloc(Module['_BinaryenSizeofLiteral']()); // a single literal in memory. the LLVM C ABI
-                                                          // makes us pass pointers to this.
```

Comment:
Nice! I think it is good to get rid of this allocated temp. Perhaps we can have a variable to store the result of `_BinaryenSizeofLiteral()`, though? It seems silly to have so many calls into a function that always returns the same value.